docs/Data/Semiring/Boolean.md

```diff
@@ -0,0 +1,28 @@
+## Module Data.Semiring.Boolean
+
+#### `BooleanSemiring`
+
+``` purescript
+newtype BooleanSemiring a
+  = BooleanSemiring a
+```
+
+Provides the semiring for any boolean algebra.
```

Comment:
Indeed... Should I provide two different newtypes? (`DisjunctiveBooleanSemiring` and `ConjunctiveBooleanSemiring`)?